TeadsSDKDemo/webviewhelper/src/main/java/tv/teads/webviewhelper/SyncAdWebView.kt

```diff
@@ -92,9 +92,9 @@ class SyncAdWebView(context: Context,
                     it.topMargin = topOffSet
                     it.bottomMargin = bottomOffSet
                 }
-            }
 
-            adLayer.addView(containerAdView)
+                addView(containerAdView)
+            }
```

Comment:
What did you changed here?

Reply:
Little improvement. It was to do all settings regarding the `adLayer` inside the `apply` block 

**Before** 

```
            val adLayer = FrameLayout(webview.context).apply {
                layoutParams = FrameLayout.LayoutParams(webview.layoutParams).also {
                    it.topMargin = topOffSet
                    it.bottomMargin = bottomOffSet
                }
            }
            
            adLayer.addView(containerAdView)
```
            
**After** 

```
            val adLayer = FrameLayout(webview.context).apply {
                layoutParams = FrameLayout.LayoutParams(webview.layoutParams).also {
                    it.topMargin = topOffSet
                    it.bottomMargin = bottomOffSet
                }

                addView(containerAdView)
            }
```